2.7.0/done/result_xpu.md

```diff
@@ -0,0 +1,109 @@
+- [Intel GPU] Avoid including CPU oneDNN header files for Intel GPU ([#147969](https://github.com/pytorch/pytorch/pull/147969))
+- [Intel GPU] Align XPU convolution_backward output layout between fake tensor and real output tensor ([#146880](https://github.com/pytorch/pytorch/pull/146880))
+- [XPU] Improve error handling and reporting in CMake files ([#149353](https://github.com/pytorch/pytorch/pull/149353))
+- [XPU] Update filter out of DG2 AOT target ([#148677](https://github.com/pytorch/pytorch/pull/148677))
+- [XPU] Refine XPU oneDNN context manager API ([#147349](https://github.com/pytorch/pytorch/pull/147349))
+- [XPU] Refine XPU external Stream ([#142347](https://github.com/pytorch/pytorch/pull/142347))
+- [XPU] Refine torch.xpu.get_device_properties API error message ([#144379](https://github.com/pytorch/pytorch/pull/144379))
+- [Intel GPU] Add XPU device to nested_layer_norm ([#148593](https://github.com/pytorch/pytorch/pull/148593))
+- [XPU] Generalize `is_big_gpu()` check in Inductor ([#143491](https://github.com/pytorch/pytorch/pull/143491))
+- [XPU] Allow coalescing path on XPU and dispatch to XPU tensor barrier if XCCL backend is specified ([#143735](https://github.com/pytorch/pytorch/pull/143735))
+
+### Bug Fixes
+- [Intel GPU] Fix SDPA dummy LSE output to match meta function ([#148652](https://github.com/pytorch/pytorch/pull/148652))
+- [Intel GPU] Fix memory leak in deconv backward ([#144385](https://github.com/pytorch/pytorch/pull/144385))
+- [Intel GPU] Fix bug: use zero-point to decide conv src zp mask ([#149473](https://github.com/pytorch/pytorch/pull/149473))
+- [XPU] Fix `TRITON_XPU_BUILD_FROM_SOURCE` ([#142850](https://github.com/pytorch/pytorch/pull/142850))
+- [XPU] Fix issue #143489 in torch.utils._content_store hash storage ([#147785](https://github.com/pytorch/pytorch/pull/147785))
+- [XPU] Fix Inductor UT failures introduced from community ([#146762](https://github.com/pytorch/pytorch/pull/146762))
+- [XPU] Fix broken XPU CI introduced by community changes ([#145058](https://github.com/pytorch/pytorch/pull/145058))
+- [XPU] Fix Inductor UT on Windows ([#146481](https://github.com/pytorch/pytorch/pull/146481))
+- [XPU] Fix missing `model_container_runner_xpu.cpp` in `libtorch_xpu.so` ([#149175](https://github.com/pytorch/pytorch/pull/149175))
+
+### Performance
+- [Intel GPU] Improve zero-point memory creation ([#148640](https://github.com/pytorch/pytorch/pull/148640))
+- [Intel GPU] Optimize qlinear.pointwise with mixed dtype support ([#136753](https://github.com/pytorch/pytorch/pull/136753))
+- [Intel GPU] Optimize qconv.pointwise with mixed dtype XPU support ([#135465](https://github.com/pytorch/pytorch/pull/135465))
+- [Intel GPU] Optimize qlinear_pointwise.binary[_tensor] XPU support ([#135337](https://github.com/pytorch/pytorch/pull/135337))
+- [Intel GPU] Optimize qconv_pointwise.binary XPU support ([#135189](https://github.com/pytorch/pytorch/pull/135189))
```

Comment:
These pull requests should be abstracted as the feature of XPU PT2E.

Reply:
done, move to feature